android/brave_java_resources.gni

```diff
@@ -637,9 +631,11 @@ brave_java_resources = [
   "java/res/drawable/ic_payment_methods.xml",
   "java/res/drawable/ic_phone.xml",
   "java/res/drawable/ic_privacy_reports.xml",
+  "java/res/drawable/ic_qr_code.xml",
```

Comment:
@wchen342 i think we are missing dark mode image here.

Reply:
Should be fixed now.